.github/format-rfcs.mjs

```diff
@@ -0,0 +1,29 @@
+import * as fs from 'node:fs/promises'
+
+const dir = new URL('../proposals/', import.meta.url)
```

Comment:
I think `active` was better. If it is in that directory, it is no longer proposed and actually to be implemented.

Reply:
The reason this uses `proposals` instead of `active` is so that a `landed` proposal does not change URLs.